contracts/Splitter.sol

```diff
@@ -24,8 +26,8 @@ contract Splitter {
     )
         external
     {
```

Comment:
should we add a 

```
        require(msg.sender == '0x00De4B13153673BCAE2616b67bf822500d325Fc3', 'only gitcoin can call')
```
to make sure only gitcoin can call the splitter contract? this prevents just anyone from being able to xfr the tokens